addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5286,7 +5286,12 @@ msgctxt "#12373"
 msgid "Settings & file manager"
 msgstr ""
 
-#empty strings from id 12374 to 12375
+#empty strings with id 12374
+
+#: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
+msgctxt "#12375"
+msgid "This will save selected values as default for all media. Changes to 'View mode' or 'Pixel ratio' affect aspect ratio on playback. Are you sure?"
```

Comment:
I'd not get specific on which settings affect the AR but rather point out that this will affect the playback of all media. Giving specific examples might be misleading in the way that people might think other values don't affect AR.